bazel/tools/java/com/code_intelligence/jazzer/tools/FuzzTargetTestWrapper.java

```diff
@@ -108,6 +108,9 @@ public static void main(String[] args) {
       if (hookJarActualPath != null) {
         command.add(String.format("--main_advice_classpath=%s", hookJarActualPath));
       }
+      command.add("--jvm_flags="
```

Comment:
Add `"-XX:+EnableDynamicAgentLoading"` here as well, as the warnings are not really helpful in our tests.